docs/src/app/components/pages/components/IconButton/ExampleComplex.jsx

```diff
@@ -1,19 +1,25 @@
 import React from 'react';
 import FontIcon from 'material-ui/lib/font-icon';
 import IconButton from 'material-ui/lib/icon-button';
+import ActionHome from 'material-ui/lib/svg-icons/action/schedule';
```

Comment:
The custom font doesn't include material-design icons, so it isn't possible to use the same icon for the three ways.

However the first way (nested `FontIcon` using the custom font) is redundant, given that `IconButton` supports the `iconClassName` prop, so we could just leave it out, and use the same material-design icon for the other two ways (SVG icon and ligature).